website/pages/blog/open-source-graphql-security.mdx

```diff
@@ -0,0 +1,242 @@
+    multifaceted world of GraphQL security. A wellspring of knowledge, it offers structured learning
+    paths, exploring vulnerabilities, attack vectors, and preventive strategies, thereby forging a
+    security-savvy developer who can intuitively construct and validate secure APIs.
+
+2.  **Best Practices**
+
+    - [9 GraphQL Security Best Practices](https://escape.tech/blog/9-graphql-security-best-practices/):
+      Dive into Escape’s comprehensive guide which unveils nine pivotal security best practices,
+      presenting a blend of actionable insights and theoretical knowledge to fortify GraphQL
+      implementations against potential threats.
+    - [The Guild’s Resource](https://the-guild.dev/blog/best-practices-of-graphql-clients): While
+      this resource by The Guild isn’t strictly security-focused, it provides invaluable best
+      practices on GraphQL clients that, when adeptly applied, augment the robustness and efficiency
+      of GraphQL APIs, subsequently enhancing their inherent security.
+
+3.  **Blogs and More**
+
+    Explore the many insights and experiences shared by experts through different channels:
+
+    - **Blogs**: Immerse yourself in rich content through blogs from
+      [Escape](https://escape.tech/blog/) and [The Guild](https://the-guild.dev/blog/), offering a
+      spectrum of perspectives, learnings, and strategies around GraphQL and cybersecurity.
+    - **Videos**: Discover visual insights through a collection of videos curated by
+      [GraphQL WTF](https://graphql.wtf/). Although not strictly centered around security,
+      understanding various facets of GraphQL enhances your capability to architect, implement, and
+      secure GraphQL APIs more effectively.
+
+Online security is always changing, often at a rapid pace. By engaging with learning platforms,
+adhering to best practices, and tapping into the collective knowledge shared through blogs and
+videos, we arm ourselves and our APIs against the multifaceted cybersecurity threats that persist in
+the digital realm.
+
+This path of continuous learning and adaptation ensures that as developers and cybersecurity
+professionals, we remain up to date to secure our GraphQL APIs against both prevalent and emerging
+threats.
+
+## Wrapping up and Joining Forces
+
+Huge shoutout to [The Guild](https://the-guild.dev/) for hosting this dive into the depths of
+GraphQL and open-source magic. Their work, and the work of countless others in our robust community,
+propels GraphQL to new heights daily. Your code, knowledge, and keen eyes are the gears moving this
+ecosystem forward.
+
+But hey, let’s not stop there! **Contribute, Engage, Elevate.** That’s the open-source mantra. Your
+expertise could well be the next big leap forward for GraphQL tools and platforms. Check out these
+awesome [GraphQL projects](https://github.com/chentsulin/awesome-graphql) and see where your skills
```

Comment:
I don't want to link to that page - better to link to graphql.org/code
awesome graphql is not well maintained and its not really ordered and there are many unmaintained resources there

Reply:
Make sense, I will add graphql.org/code & maybe https://github.com/Escape-Technologies/awesome-graphql-security ?

Reply:
I forgot this change, adding it. Tho I agree awesome graphql is not really up to date, but neither is https://graphql.org/code/ + it's really not accessible ://

Reply:
great